paymentsheet/src/test/java/com/stripe/android/link/LinkActivityViewModelTest.kt

```diff
@@ -554,7 +554,7 @@ internal class LinkActivityViewModelTest {
         assertThat(appBarState.email).isNull()
         assertThat(appBarState.showHeader).isFalse()
         assertThat(appBarState.showOverflowMenu).isFalse()
-        assertThat(appBarState.navigationIcon).isEqualTo(R.drawable.stripe_link_back)
+        assertThat(appBarState.navigationIcon).isEqualTo(R.drawable.stripe_link_close)
```

Comment:
Expected, as the user can't go back at this point. They can only close the Link UI.